cmd/osmosisd/cmd/root.go

```diff
@@ -110,9 +110,17 @@ func initAppConfig() (string, interface{}) {
 ###############################################################################
 
 [osmosis-mempool]
+# This is the max allowed gas any tx. 
+# This is only for local mempool purposes, and thus	is only ran on check tx.
+max-gas-wanted-per-tx = "25000000"
+
 # This is the minimum gas fee any arbitrage tx should have, denominated in uosmo per gas
 # Default value of ".005" then means that a tx with 1 million gas costs (.005 uosmo/gas) * 1_000_000 gas = .005 osmo
 arbitrage-min-gas-fee = ".005"
+
+# This is the minimum gas fee any tx with high gas demand should have, denominated in uosmo per gas
+# Default value of ".01" then means that a tx with 1 million gas costs (.01 uosmo/gas) * 1_000_000 gas = .01 osmo
+min-gas-price-for-high-gas-tx = ".01"
```

Comment:
```suggestion
# Default value of ".005" then means that a tx with 1 million gas costs (.005 uosmo/gas) * 1_000_000 gas = .005 osmo
min-gas-price-for-high-gas-tx = ".005"
```

Reply:
to be consistent with arb gas

Reply:
ah, copy-paste :)

Reply:
will fix a test quickly

Reply:
tysm!

Also no worries, didn't specify anything about concrete number to default to (or think about it too much before)

Reply:
test fixed, commits squashed